src/index.test.js

```diff
@@ -17,23 +31,46 @@ describe('cmp.init', () => {
 
 	it('inititalises CCPA when in the US', () => {
 		cmp.init({ isInUsa: true });
-		expect(initCCPA).toHaveBeenCalledTimes(1);
+		expect(CCPA.init).toHaveBeenCalledTimes(1);
 	});
 
 	it('inititalises TCF when not in the US', () => {
 		cmp.init({ isInUsa: false });
-		expect(initTCF).toHaveBeenCalledTimes(1);
+		expect(TCFv2.init).toHaveBeenCalledTimes(1);
 	});
```

Comment:
Would it be worth adding a test for multiple `init()` calls?

Possible duplicate of https://github.com/guardian/consent-management-platform/commit/eba15aedc6906a9d1431457c03717ca8da01f7b3#r40481144